fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -93,13 +106,17 @@ async def recv(self):
     async def close(self, code: int = 1000):
         if self._websocket is not None:
             # Case opened, we have something to close.
-            self._websocket.close(code)
+            await asyncio.get_event_loop().run_in_executor(
+                None, self._websocket.close, code
```

Comment:
if this is the same _websocket as in WebSocketsClientHandler, isn't at an async function already ?